gradio/chat_interface.py

```diff
@@ -96,9 +100,8 @@ def __init__(
             textbox: an instance of the gr.Textbox or gr.MultimodalTextbox component to use for the chat interface, if you would like to customize the textbox properties. If not provided, a default gr.Textbox or gr.MultimodalTextbox component will be created.
             additional_inputs: an instance or list of instances of gradio components (or their string shortcuts) to use as additional inputs to the chatbot. If components are not already rendered in a surrounding Blocks, then the components will be displayed under the chatbot, in an accordion.
             additional_inputs_accordion: if a string is provided, this is the label of the `gr.Accordion` to use to contain additional inputs. A `gr.Accordion` object can be provided as well to configure other properties of the container holding the additional inputs. Defaults to a `gr.Accordion(label="Additional Inputs", open=False)`. This parameter is only used if `additional_inputs` is provided.
-            examples: sample inputs for the function; if provided, appear below the chatbot and can be clicked to populate the chatbot input. Should be a list of strings if `multimodal` is False, and a list of dictionaries (with keys `text` and `files`) if `multimodal` is True.
             cache_examples: if True, caches examples in the server for fast runtime in examples. The default option in HuggingFace Spaces is True. The default option elsewhere is False.
-            examples_per_page: if examples are provided, how many to display per page.
+            examples: A list of examples messages to display in the chatbot. The `main_text` field is required, and the `additional_input` field is optional. The `additional_input` field can be a string or a `FileMessage` object.
```

Comment:
This documentation looks incorrect. (See my comment below on chatbot.py). For backwards compatibility, can we also accept a list of strings, and just convert them to a list of dicts with the "text" keys?

Reply:
I think it would be good to only accept dicts as it would prep users for when "messages" is the only allowed type

Reply:
Agreed, not sure we still want to support just strings anymore.

Reply:
Ok I don't feel too strongly, but I would then just have a special error if a user passes in a list of strings telling them that format has been deprecated and they have to now pass in a list of dictionaries with key "text". Because I'm sure this will trip up users who are migrating to 5.0